<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Monitor Jakości Powietrza</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainVerticalLayout">
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>Odswież</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="cityFilterEdit">
        <property name="placeholderText">
         <string>Wpisz miasto</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filterButton">
        <property name="text">
         <string>Filtruj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="mapButton">
        <property name="text">
         <string>Mapa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="leftColumn">
        <item>
         <widget class="QListWidget" name="stationList"/>
        </item>
        <item>
         <widget class="QListWidget" name="sensorList"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="logBrowser">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="rightColumn">
        <item>
         <widget class="QTextBrowser" name="measurementBrowser"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="airQualityBrowser"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
